File: Patterns/printPattern2.cpp
#include <iostream>
using namespace std;
void pattern2(int row){
    int nst=row;
    for(int r=1; r<=row; r++){
        for(int cst=1; cst<=nst; cst++){
            cout<<"*\t";
        }
        nst--;
        cout<<endl;
    }
}
int main(int argc, char **argv){
    int n;
    cin >> n;
    pattern2(n);
    //write your code here
    
}

File: Patterns/printPattern2.java
import java.util.*;

public class Main {
    public static void printPattern2(int row) {
        int nst = row;
        for (int r = 1; r <= row; r++) {
            for (int cst = 1; cst <= nst; cst++) {
                System.out.print("*\t");
            }
            nst--;
            System.out.println();
        }
    }

    public static void main(String[] args) {
        Scanner scn = new Scanner(System.in);
        int row = scn.nextInt();
        printPattern(row);

        // write ur code here

    }
}